src/accessibility/describe.js

```diff
@@ -80,41 +80,40 @@ p5.prototype.describe = function(text, display) {
   text = _descriptionText(text);
   //if there is no dummyDOM
   if (!this.dummyDOM) {
-    this.dummyDOM = document.getElementsByTagName('body')[0];
+    this.dummyDOM = document.getElementById(cnvId).parentNode;
+  }
+  if (!this.descriptions) {
+    this.descriptions = {};
   }
   //check if html structure for description is ready
-  if (this.dummyDOM.querySelector(`#${cnvId}${fallbackDescId}`)) {
+  if (this.descriptions.fallback) {
     //check if text is different from current description
-    if (
-      this.dummyDOM.querySelector(`#${cnvId}${fallbackDescId}`).innerHTML !==
-      text
-    ) {
+    if (this.descriptions.fallback.innerHTML !== text) {
       //update description
-      this.dummyDOM.querySelector(
-        `#${cnvId}${fallbackDescId}`
-      ).innerHTML = text;
+      this.descriptions.fallback.innerHTML = text;
     }
   } else {
     //create fallback html structure
-    let results = _describeFallbackHTML(this.dummyDOM, cnvId, text);
-    this.dummyDOM = results;
+    let results = _describeFallbackHTML(this.dummyDOM, cnvId);
+    this.descriptions.fallback = results[0];
+    this.dummyDOM = results[1];
+    this.descriptions.fallback.innerHTML = text;
   }
   //if display is LABEL
   if (display === this.LABEL) {
     //check if html structure for label is ready
-    if (this.dummyDOM.querySelector(`#${cnvId}${labelDescId}`)) {
+    if (this.descriptions.label) {
       //check if text is different from current label
-      if (
-        this.dummyDOM.querySelector(`#${cnvId}${labelDescId}`).innerHTML !==
-        text
-      ) {
+      if (this.descriptions.label.innerHTML !== text) {
         //update label description
-        this.dummyDOM.querySelector(`#${cnvId}${labelDescId}`).innerHTML = text;
+        this.descriptions.label.innerHTML = text;
       }
     } else {
       //create label html structure
-      let results = _describeLabelHTML(this.dummyDOM, cnvId, text);
-      this.dummyDOM = results;
+      let results = _describeLabelHTML(this.dummyDOM, cnvId);
+      this.descriptions.label = results[0];
+      this.dummyDOM = results[1];
```

Comment:
same question here as above -- why does this overwrite setting `this.dummyDOM` on line 83?